### PR TITLE
Create, edit, delete, exchange rate currency

### PR DESCRIPTION
**Pull request by ansar21mallouli**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Create, edit, delete and live exchange rates currency
| Type?         | new feature
| Category?     |  TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | TEST_PATH=full/11_international/1_localization/3_currency npm run specific-test -- --URL=URLFrontOffice

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11069)

### CODE REVIEW

File: tests/E2E/test/campaigns/common_scenarios/currency.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    scenario('Edit and cancel "Currency"', client => {
+      test('should click on "Edit" action from the dropdown list', () => client.clickOnAction(Localization.Currencies.edit_button));
+      test('should click on "Cancel" button', () => client.waitForExistAndClick(Localization.Currencies.cancel_button));
+    }, 'currency');
+  },
+
+  deleteCurrency(confirmDelete, successMessage = true, message = '') {
+    scenario('Delete "Currency"', client => {
+      test('should click on "Delete" action from dropdown list', () => client.clickOnAction(Localization.Currencies.delete_action_button, Localization.Currencies.action_button, 'delete', confirmDelete));
+      if (successMessage) {
```
Comment (fatmaBouchekoua) [inline tests/E2E/test/campaigns/common_scenarios/currency.js]: You can use only the message parameter

**Review by fatmaBouchekoua — CHANGES_REQUESTED** [review]
Almost good ! Only one change.

**Comment by ansar21mallouli** [comment]
@fatmaBouchekoua : Done.

**Review by saymouMb — DISMISSED** [review]

**Review by fatmaBouchekoua — DISMISSED** [review]

**Comment by marionf** [comment]
@ansar21mallouli

I don't know why but there are some steps that take a lot of time:

1) Check the sort of currencies by iso code: 
- should click on sort by ASC icon
- should click on sort by desc icon

2) Check the sort of currencies by exchange rate
- should click on sort by ASC icon
- should click on sort by desc icon

3) delete currency
- should click on delete action from dropdown

Can we do it faster ?

**Comment by ansar21mallouli** [comment]
@marionf 
For me the launch of the entire test sequence (create, edit, delete and exchange rate currency) takes 2 minutes.
The steps that you specify in your last comment take a few time to be executed.
In your case, how long it takes for you to execute all steps ?

**Comment by marionf** [comment]
The entire sequence takes 13 minutes for me :hushed:

I made a screenrecord: https://drive.google.com/file/d/1hXFH5ssW70lc7aT68Qzzs-V2eH2zqfpS/view?usp=sharing

**Comment by ansar21mallouli** [comment]
@marionf 
To run the test you need a fresh installation with English language & French country.

**Comment by marionf** [comment]
All right, it's better like this :)
Thank you @ansar21mallouli

**Review by fatmaBouchekoua — DISMISSED** [review]

**Review by fatmaBouchekoua — DISMISSED** [review]

**Comment by fatmaBouchekoua** [comment]
@mbadrani this PR is ready to be merged

**Review by fatmaBouchekoua — APPROVED** [review]

**Pull request closed without merging**